crates/pipeline_manager/migrations/V1__add_pipeline_runtime_status.sql

```diff
@@ -0,0 +1,32 @@
+-- Store runtime state of pipelines in a separate table.
+-- DB schema support for desired-state-based pipeline
+-- management.
+
+CREATE TABLE IF NOT EXISTS pipeline_runtime_state (
+    id uuid PRIMARY KEY,
+    tenant_id uuid NOT NULL,
+    -- TODO: add 'host' field when we support remote pipelines.
+    port smallint,
```

Comment:
Can we already make this field opaque via a varchar (I suggest calling it location)? We can then have a runner-specific naming scheme to find where the entry is. I could also add the change myself but that'd be another migration.

Reply:
Fair enough, I can do it.